Jet/src/test/java/org/mitre/jet/ebts/EbtsParserTest.java

```diff
@@ -129,4 +131,31 @@ public void testDescriptiveOnly() throws Exception {
 
     }
 
-}
+    @Test
+    public void typeEmptyImage10Test() throws Exception {
+        File file = new File(ClassLoader.getSystemResource("EFT/empty_image.eft").toURI());
+
+        EbtsParser ebtsParser = new EbtsParser();
+        //previously threw exception
+        Ebts ebts = ebtsParser.parse(file,ParseType.FULL);
+
+        assertNotNull(ebts);
+        GenericRecord type10 = (GenericRecord) ebts.getRecordsByType(10).get(0);
+        assertFalse(type10.getFields().isEmpty());
+        assertEquals("JPEGB",type10.getField(11).toString());
+        //TODO if we want this to be false,  hasImageData needs to be updated to include getField(dataField).getData().length > 0
+        assertTrue(type10.hasImageData());
+    }
+
+    @Test
+    public void type7ImageBoundsTest() throws Exception {
+        File file = new File(ClassLoader.getSystemResource("EFT/type7_image_oob.eft").toURI());
```

Comment:
@jej2003 Can you add type7_image_oob.eft?

I'm going to comment it out in the interim and merge what the rest.